packages/react/src/components/DropdownMenu/DropdownMenuList.tsx

```diff
@@ -0,0 +1,19 @@
+import cl from 'clsx/lite';
+import { forwardRef } from 'react';
+import type { HTMLAttributes } from 'react';
+
+export type DropdownMenuListProps = HTMLAttributes<HTMLUListElement>;
+
+export const DropdownMenuList = forwardRef<
+  HTMLUListElement,
+  DropdownMenuListProps
+>(function DropdownMenuGroup({ className, ...rest }, ref) {
+  return (
+    <ul
+      ref={ref}
+      role='group'
+      className={cl('ds-dropdownmenu__group', className)}
```

Comment:
Ser ut som det henger igjen en `DropdownMenuGroup` og `ds-dropdownmenu__group` her.
Lurer også på hvorfor vi ikke extender `List.Unordered` (som fikser litt UU-issues) og hvorfor vi setter `role="group"`? :)

Reply:
Sant det! den gjør litt greier vi ikke vil ha. Har faktisk vurdert om vi skulle hatt `List.Menu` som gjør lite endringer. ListUnordered fikser slik at dersom man endrer på `display` eller setter `list-style: none`, så forstår Safari fortsatt at dette er en liste (den mister liste-rolle på Safari når man endrer på CSS som er helt håpløst.. https://adrianroselli.com/2018/05/functions-to-add-aria-to-tables-and-lists.html)

Reply:
Så skal me resolve denne for no, og ta det opp igjen på eit seinare tidspunkt?

Reply:
Eg tok også vekk group rollen, den hang igjen fra gammalt av med rolle menu

Reply:
jupp - støtter at vi kan ta det siden ifbm. UU-gjennomgangen ^^